politeiad/politeiad.go

```diff
@@ -866,7 +866,7 @@ func _main() error {
 	// initializes logging and configures it accordingly.
 	loadedCfg, _, err := loadConfig()
 	if err != nil {
-		return fmt.Errorf("Could not load configuration file: %v", err)
```

Comment:
This error message is helpful when the error != flags.ErrHelp.  I would prefer that you didn't get rid of it.  

A different way to do this might be to check if `Parse()` returned `flags.ErrHelp` in the config file.  If it did, you can print the help message and exit.  If it returned a different error than that means there was some issue with parsing the config file, in which case you would want to return the error like normal so that `Could not load configuration file: some parsing error` would be printed to stdout.

Reply:
```
	// Pre-parse the command line options to see if an alternative config
	// file or the version flag was specified.  Any errors aside from the
	// help message error can be ignored here since they will be caught by
	// the final parse below.

```

Actually, according to this comment you can just ignore any errors that are not `ErrHelp` during the pre-parse.  If it is `ErrHelp` you can just print the help message and exit instead of returning the error.

Reply:
Hey @lukebp thanks for the review. You're right, the solution was selfcontained in the comment hahaha didn't get to notice that. So now, if the error returned from the pre-parse operation is equal to `ErrHelp`, we print the help message and exit os instead of returning the error down the execution line, which was causing the double printing when it got to `main()`.